client/my-sites/invites/controller.js

```diff
@@ -55,7 +55,11 @@ export function acceptInvite( context, next ) {
 				debug( 'Accepted invite for VIP sites' );
 				window.location.href = getRedirectAfterAccept( acceptedInvite );
 			} else {
-				const redirect = getRedirectAfterAccept( acceptedInvite );
+				const redirect = getRedirectAfterAccept(
+					acceptedInvite,
+					get( acceptedInvite, 'site.is_wpforteams_site', false )
```

Comment:
could we avoid passing the second argument? since we already have all the Data we need inside the "invite"?

Reply:
Great catch, updated!